README.md

```diff
@@ -315,6 +315,22 @@ $query->setQuery('search term');
 $adapter = new SolariumAdapter($solarium, $query);
 ```
 
+### SphinxAdapter
+To paginate a [Sphinx](http://php.net/manual/en/class.sphinxclient.php) query:
```

Comment:
Is there a better page you could link to?  The PHP client page doesn't actually say what Sphinx is.  Is it http://sphinxsearch.com/?